globus_cli/commands/delete.py

```diff
@@ -38,14 +39,16 @@ def delete_command(batch, ignore_missing, recursive, endpoint_plus_path,
             'delete requires either a PATH OR --batch')
 
     client = get_client()
-    autoactivate(client, endpoint_id, if_expires_in=60)
+    autoactivate(client, endpoint_id, if_expires_in=60,
+                 skip_failure=skip_activation_check)
```

Comment:
What's the rationale behind still doing autoactivation if skip activation is set?

Reply:
I was thinking "just in case the endpoint can be autoactivated, why not try?"

The answer to my question is "because it takes another round trip on the network, and no-one will use this on autoactivatable endpoints"

I'll take this out.

Reply:
I'm not super concerned with the extra request, I was more curious if there was still some reason we needed to call autoactivate.

But yeah, I think dropping it if we can is the way to go. It's confusing if nothing else.